src/main/java/me/ryanhamshire/GriefPrevention/PlayerEventHandler.java

```diff
@@ -1719,7 +1720,9 @@ else if( clickedBlock != null &&
 				clickedBlockType == Material.BIRCH_FENCE_GATE    ||
 				clickedBlockType == Material.JUNGLE_FENCE_GATE   ||
 				clickedBlockType == Material.SPRUCE_FENCE_GATE   ||
-				clickedBlockType == Material.DARK_OAK_FENCE_GATE)))
+				clickedBlockType == Material.DARK_OAK_FENCE_GATE)) ||
+		(instance.config_claims_preventReadingLecterns && (
+				clickedBlockType == Material.LECTERN)))
```

Comment:
Is this necessary? I was under the assumption that players cannot read from lecterns in the first place since that's considered a /containertrust interaction by GP already.

Reply:
Yeah, the change on line 1630 above removes the container trust requirement, and this line adds access trust requirement (if config set to true)